etc/cc-dev.conf

```diff
@@ -26,9 +21,6 @@ server {
     proxy_pass         http://localhost:7781/login;
     proxy_redirect     off;
 
-    proxy_set_header   Host             $host;
-    proxy_set_header   X-Real-IP        $remote_addr;
-    proxy_set_header   X-Forwarded-For  $proxy_add_x_forwarded_for;
     proxy_max_temp_file_size 0;
 
     client_max_body_size       10m;
```

Comment:
Let's not cargo-cult this stuff either.

Reply:
The defaults for most of these settings seems fine, so I've deleted a bunch of them.